api/programs/programTypesModel.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../../data/db-config');
+
+const getAll = async () => {
+  return db('programs');
+};
+
+const getById = async (id) => {
+  return db('programs').where('program_id', id).first();
+};
+
+const getByName = async (program_name) => {
```

Comment:
I like this model function, I can see how it might be useful for the frontend to be able to fetch by name and by ID. Good stuff!